cmake/Modules/FindQhull.cmake

```diff
@@ -9,14 +9,14 @@
 # If QHULL_USE_STATIC is specified then look for static libraries ONLY else
 # look for shared ones
 
-set(QHULL_MAJOR_VERSION 6)
+set(QHULL_MAJOR_VERSION 8)
 
 if(QHULL_USE_STATIC)
   set(QHULL_RELEASE_NAME qhullstatic)
```

Comment:
I think this should be changed to `qhullstatic_r` (and below `qhullstatic_r_d`)

Reply:
fixed, thanks